pkgs/tools/system/proot/default.nix

```
@@ -19,8 +24,13 @@ stdenv.mkDerivation {
     sed -i /CROSS_COMPILE/d src/GNUmakefile
   '';
 
-  buildInputs = [ talloc ] ++ lib.optional enablePython python;
-  nativeBuildInputs = [ docutils ] ++ lib.optional enablePython swig;
+  buildInputs = [ ncurses libarchive talloc ] ++ lib.optional enablePython python3;
+  nativeBuildInputs = [ pkg-config docutils ] ++ lib.optional enablePython swig;
+  patches = [
+    # without this patch the package does not build with python>3.7
+    (fetchpatch { url = "https://github.com/proot-me/proot/pull/285.patch";
+                  sha256= "1vncq36pr4v0h63fijga6zrwlsb0vb4pj25zxf1ni15ndxv63pxj"; })
+];
```

Can we do a more conventional indentation
```suggestion
    (fetchpatch {
      url = "https://github.com/proot-me/proot/pull/285.patch";
      sha256= "1vncq36pr4v0h63fijga6zrwlsb0vb4pj25zxf1ni15ndxv63pxj";
    })
  ];
```